test/fancylogger.py

```diff
@@ -51,6 +52,24 @@ def classless_function():
     logger.warn("from classless_function")
 
 
+class FancyLoggerLogToFileTest(EnhancedTestCase):
+    """
+    Tests for fancylogger, specific for logToFile
+    These dont' fit in the FancyLoggerTest class because they don't require the setUp and tearDown
```

Comment:
they don't *require* it, but do they affect this test at all?

If not, please include `test_logtofile` in `FancyLoggerTest` instead, since this is a style-break (we usually only have one class per test module).